src/Standard/Exception/NotImplementedError.php

```diff
@@ -9,6 +9,8 @@
 /** @internal */
 final class NotImplementedError extends RuntimeException
 {
-    /** @var string */
-    protected $message = 'An operation is not implemented.';
+    public function __construct(string|null $message = null)
```

Comment:
I'd prefer to have the previous version. Remove this boilerplate 👍🏼

Reply:
phpcs keeps recommending to type the parameter:
```
 13 | ERROR | [x] Property \Codelicia\Niltok\Standard\Exception\NotImplementedError::$message does not have native type hint for its value but it
    |       |     should be possible to add it based on @var annotation "string".
```

but that is not really possible:
```
+Fatal error: Type of Codelicia\Niltok\Standard\Exception\NotImplementedError::$message must not be defined (as in class Exception) in .../github.com/codelicia/niltok/src/Standard/Exception/NotImplementedError.php on line 10
```

Reply:
I will remove the check.